build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.1'
+        classpath 'com.android.tools.build:gradle:2.1.0-beta1'
```

Comment:
I know these are Beta samples, but would prefer users only have to deal with one set of Beta issues at a time, so personally I'd think we should use version 2.0.0 here (users always have option to move to beta versions of gradle/android studio themselves).